homeassistant/components/config/automation.py

```diff
@@ -22,9 +22,10 @@ async def async_setup(hass: HomeAssistant) -> bool:
 
     async def hook(action: str, config_key: str) -> None:
         """post_write_hook for Config View that reloads automations."""
-        await hass.services.async_call(DOMAIN, SERVICE_RELOAD)
-
```

Comment:
This could be in a separate PR instead; there's no need to reload when deleting an automation